app/src/main/java/org/p2p/wallet/jupiter/repository/tokens/JupiterSwapTokensRemoteRepository.kt

```diff
@@ -67,4 +72,22 @@ internal class JupiterSwapTokensRemoteRepository(
         val now = System.currentTimeMillis()
         return (now - fetchTokensDate) <= TimeUnit.DAYS.toMillis(1) // check day has passed
     }
+
+    override suspend fun getTokenRate(token: JupiterSwapToken): TokenPrice? =
```

Comment:
![Screenshot 2023-03-12 at 23 29 50](https://user-images.githubusercontent.com/122359284/224561832-e8bbd1d2-e4ce-4bf0-9ac4-0a6dc3566933.png)
You can unwrap  from dispatcher(io) cause our repository calls already wrapped into dispatchers

Reply:
done, thanks!